app/views/sales.py

```diff
@@ -41,21 +41,17 @@ def post(self):
         return {
             'message': 'Sale has been created successfully.', 'sale': sale
         }, 201
-    def get(self, sale_id=None):
+    @classmethod
+    def get(cls, sale_id=None):
         '''Get sales.'''
 
         if sale_id:
             sale = Sale.get(id=sale_id)
             if sale:
-                return {
-                        'message': 'Sale record found.', 'sale': sale.view()
-                    }, 200
+                return {'message': 'Sale record found.', 'sale': sale.view()}, 200
 
             return {'message': 'Sale record not found.'}, 404
 
         sales = Sale.get_all()
         sales = [sales[sale].view() for sale in sales]
         return {'message': 'Sales records found.', 'sales': sales}, 200
```

Comment:
Presenting your get request in [this manner](https://user-images.githubusercontent.com/22760022/47084867-84e0a700-d21d-11e8-908f-1f85e668b17f.png) is not the best way. It's redundant e.g. you're repeating your id value.

it should be of the form:-
```
{
  "message": "Sales records found.",
  "sales": [
    {"id": 1, "name": "product 1", "price": 7809},
    {"id": 2, "name": "product 2", "price": 789}
  ]
}
```

Note that I used a list since in one sale a customer can purchase multiple items.

Reply:
Go through [this](https://labs.omniti.com/labs/jsend) to see some json response conventions you can adopt for your App.

Reply:
When am presenting my get request of sales.The 1st ID is the sale ID and the 2nd one is the product ID is for the product being sold.

> Presenting your get request in [this manner](https://user-images.githubusercontent.com/22760022/47084867-84e0a700-d21d-11e8-908f-1f85e668b17f.png) is not the best way. It's redundant e.g. you're repeating your id value.
> 
> it should be of the form:-
> 
> ```
> {
>   "message": "Sales records found.",
>   "sales": [
>     {"id": 1, "name": "product 1", "price": 7809},
>     {"id": 2, "name": "product 2", "price": 789}
>   ]
> }
> ```
> 
> Note that I used a list since in one sale a customer can purchase multiple items.

Reply:
Okay lemme work on testing the length of response when fetching all.

Reply:
I see. Makes sense. Make a distinction between the `sales id` and `products id`

Reply:
Cool lemme do that.I have implemented fetching all feedback.